doc/source/reference/c-api.ufunc.rst

```diff
@@ -93,12 +93,22 @@ Functions
         the corresponding 1-d loop function in the func array.
 
     :param types:
-        Must be of length (*nin* + *nout*) \* *ntypes*, and it
-        contains the data-types (built-in only) that the corresponding
-        function in the *func* array can deal with.
+
+        Char of length ``(nin + nout) * ntypes`` It encodes the
```

Comment:
Char array would be clearer. capitalizing `Char` looks weird too.

Could take advantage of C 2d arrays here, and write this as ``` ``char[ntypes][nin+nout]`` array ```

Reply:
Like it in principle, but as this is not done anywhere in the code, it may lead to confusion...

Reply:
I suppose I'll have to make a patch at some point to make it so we do do that in the code.

Reply:
That would definitely make the definitions a lot clearer!